build-tools/build-deb.sh

```diff
@@ -7,7 +7,14 @@ if [ -z ${1+x} ]; then
 fi
 
 BINARY_PATH="$1"
-VERSION=$("${BINARY_PATH}" version 2>&1)
+# It's needed for creating an RC deb package
```

Comment:
My heart bleeds when I realize that versions of a binary and the deb package it belongs can be different.